src/Microsoft.VisualStudio.Editors/PropPages/xlf/compileproppage2.zh-Hans.xlf

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
-  <file datatype="xml" source-language="en" target-language="zh-Hans" original="../compileproppage2.resx">
+  <file datatype="xml" source-language="en" target-language="zh-HANS" original="../compileproppage2.resx">
```

Comment:
Why was this cased changed? Every other file has this as "zh-Hans".

Reply:
This will be fixed by updating PR